src/d3a/d3a_core/export.py

```diff
@@ -71,15 +71,15 @@ def export_json_data(self, directory: dir):
         mkdir_from_str(json_dir)
         kpi_file = os.path.join(json_dir, "KPI")
         with open(kpi_file, 'w') as outfile:
-            json.dump(self.kpi.performance_index, outfile)
+            json.dump(self.kpi.performance_index, outfile, indent=2)
```

Comment:
Not a problem at all to have 2-space indentation for JSON files, so either way if you prefer this format keep it, but out of curiosity was there a need/request/reason?

Reply:
This is just for better readability of the aggregated_result files.
Before, the results were just in one line.

Reply:
I see, very nice! Thanks for explaining